src/transform/src/lib.rs

```diff
@@ -168,15 +168,6 @@ impl<'a> TransformArgs<'a> {
         }
     }
 
-    /// Generates a `TransformArgs` instance for the given `IndexOracle` with a `GlobalId`
-    pub fn with_id(indexes: &'a dyn IndexOracle, global_id: &'a GlobalId) -> Self {
-        Self {
-            indexes,
-            stats: &EmptyStatisticsOracle,
-            global_id: Some(global_id),
-        }
-    }
-
```

Comment:
Stray?

Reply:
I wanted to stealth-push this "technical debt" in this PR (it removes an unneeded method).